### PR TITLE
Big test refactoring + Rename "nameSpace" to "namespace" + Imrove readme + And more...

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ✔️
| Fixed issues  | -

- Add ID in table "migration"
- Remove check empty DSN in `ListTableService`
- Rename `nameSpace` to `namespace`
- Remove folders `migration` and `runtime`
- Improve readme
- Total tests refactoring
- More tests

### CODE REVIEW

File: src/Service/MigrationService.php [hunk 1]
```diff
@@ -170,33 +155,12 @@ public function getMigrationHistory(?int $limit = null): array
...
-            } else {
-                $row['canonicalVersion'] = $row['version'];
-            }
-
-            $row['apply_time'] = (int) $row['apply_time'];
+            $row['apply_time'] = (int)$row['apply_time'];
             $history[] = $row;
         }
 
-        usort($history, static function (array $a, array $b) {
```
Comment (samdark) [inline src/Service/MigrationService.php:184]: Is that not needed anymore?

File: src/Service/MigrationService.php [hunk 2]
```diff
@@ -170,33 +155,12 @@ public function getMigrationHistory(?int $limit = null): array
                 continue;
             }
 
-            if (preg_match('/m?(\d{6}_?\d{6})(\D.*)?$/is', $row['version'], $matches)) {
```
Comment (samdark) [inline src/Service/MigrationService.php:173]: Is that not needed anymore?

File: src/Service/MigrationService.php [hunk 3]
```diff
@@ -450,7 +414,8 @@ private function createMigrationHistoryTable(): void
         $this->consoleHelper->io()->section("Creating migration history table \"$tableName\"...");
 
         $this->db->createCommand()->createTable($this->migrationTable, [
-            'version' => 'varchar(' . $this->maxNameLength . ') NOT NULL PRIMARY KEY',
+            'id' => 'pk',
```
Comment (samdark) [inline src/Service/MigrationService.php:417]: Why?

Comment (vjik) [reply src/Service/MigrationService.php:417]: For example, we have several migration and run migration. Migration fast and applied by one second: `apply_time` equal. Migration history can return migrations with incorrect orderd. `id` resolve this problem.

File: src/Service/MigrationService.php (hunk 2 above)
Comment (vjik) [reply src/Service/MigrationService.php:173]: Not need. Now use `id` for correct order.

File: src/Service/MigrationService.php (hunk 1 above)
Comment (vjik) [reply src/Service/MigrationService.php:184]: Not need. Now use id for correct order.

**Comment by samdark** [comment]
Thanks.

**Pull request merged**